plugin/src/main/java/org/opensearch/ml/utils/IndexUtils.java

```diff
@@ -134,4 +138,23 @@ public void getNumberOfDocumentsInIndex(
         }
     }
 
+    public static SearchRequest buildHiddenModelSearchRequest() {
```

Comment:
Is this the right place to build the searchRequest? This IndexUtils is only for Index related method. Why not put this in the RestActionUtils?

Reply:
well this is also index related search request. I thought of doing in `RestActionUtils` then felt like it can go here.